lib/menu.csv

```diff
@@ -0,0 +1,5 @@
+pizza_1,1
```

Comment:
I don't really know that much about CSV files so this is sort of for my curiosity  - how come you stored the menu as a CSV rather than a hash?

Reply:
i decided to ~~steal from~~ build on functionality we made in the stu_dir, thought that it could be more versatile/less dependant code if different menus could be loaded in